### ISSUE TITLE
competitive analysis 

### ISSUE THREAD

**Issue by Andykmcc** [Andykmcc/project-graph#32]
**old garb:**

- https://www.atlassian.com/software/jira
- https://www.pivotaltracker.com/
- http://www.redmine.org/projects/redmine

**new kids on the block:**

- https://www.zenhub.com/product
- https://waffle.io/
- https://codetree.com/
- https://zube.io/

**Comment by perryh** [comment]
trello

**Comment by mikeswoods** [comment]
* https://www.atlassian.com/software/jira 
  * We all know and loathe this
* https://www.pivotaltracker.com
  * I've used this; Pivot labs isn't a software company, rather a consultancy. PT is ok at what it is: a buy-in funnel for Agile planning. if all you ever want is Agile, then PT is your ticket.
* http://www.redmine.org/projects/redmine
  * Used it years ago. Very bare-bones. It's more-or-less a poor man's JIRA.
* https://www.zenhub.com/product
  * Seems to be in the analytics space and a clone of the major features of Trello
* https://waffle.io/
  * More card-based organization. This is a problem we are trying to overcome: software need not be based on real world analogs (desktop, file, card, i.e. anything skeuomorphic)
 * https://codetree.com/
   * More github integration and very PT-like
* https://zube.io/
  * More agile worship. And a Pivotal Tracker knockoff to boot

@Andykmcc I think you're right. The space is open to offer something new. The efforts based model data organization is interesting, in that it is simple and "open." It may be more suited to conceptualization rather than organization, but we can't really know that yet. 

I think the key for this project is _how_ to 1) present data to the user and 2) meaningfully derive insight (not necessarily in the space of analytics) from it.

**Comment by Andykmcc** [comment]
i agree with all your takes on the above companies/products. I think once we have a basic underlying framework in place to build projects on top of one feature that could be easy is the ability to load a PM philosophy on top of it. aka, you do agile and want those features, okay. just click this button and you'll have epics and stories and sprints etc. but that is kind of an up sell later. In terms in insight i think the graph model enabled us to answer a bunch of question that in other platforms need to be inferred. 

for example, when there is a feature that needs to be made and few parties (engineers, designers, managers, qa) sitting down to figure it out. a Product manger lays out the vision for _what_ they are building. "we need a widget that does A, B and C by this date and this is out team." then everyone thinks about if from their perspective and starts throwing their ideas on wall/board. at that point you just kind of have this unordered list things that need to happen. no big, some small, some isolated, some interdependent. then every tried to estimate roughly how big each thing is they said. then the group runs a kind of "sort" on it to get everything roughly at the same scale. this might include grouping things or sub tasking items. then you figure out what depends on what. all these task are discrete and really only need to consider what is immediately at hand.

From there, the software can then query the graph and tell you things like, based interdependencies this is the order things need to happen. based on estimates this is the minimum amount of time needed to get this done assuming maximum parallelization of work. it can just build your gantt charts, timelines, etc... for you without anyone really needed to things on a big picture level. this is useful because it means all the thinking is about concrete deliverables, not abstractions performed by other people.
